### ISSUE TITLE
Ability to compare Location objects

### ISSUE THREAD

**Issue by wsilva32** [dronekit/dronekit-python#321]
It would be great to be able to compare location objects or equality (or inequality).

Example:

a = Location(1.0,2.0,3.0)
b = Location(1.0,2.0,3.0)
>>> a = b
True

**Comment by hamishwillee** [comment]
I'm not sure how useful that would be unless we could also add in a "tolerance" to the comparison. I can see a situation where you're at the same space within the tolerance of your GPS signal but the comparison still isn't matching exactly.

**Comment by wsilva32** [comment]
Good point, a tolerance may be required for such an ability. It would be useful for performing unit testing for any code generating or reading location objects (rather than running comparisons on each lat,lon,alt individually). Unit testing does include an "assertAlmostEqual" but I'm not sure if that would work with object comparisons.

**Comment by tcr3dr** [comment]
For the unit testing usecase, I support the change; in practice it's doubtful anyone would try equality otherwise.

Nominating for 2.0

**Comment by hamishwillee** [comment]
Longer term this might need to change again. I'd quite like Location objects to respect a specified frame rather than return the boolean isRelative.
